resources.tf

```diff
@@ -1,3 +1,5 @@
+# KMS Key Creation
```

Comment:
I'm not a big fan of this kind of comment. It's quite clear what the code is aiming to do :)

I'd rather split `resources.tf` into things like `s3.tf` (including the associated KMS key), `security_groups.tf`, etc. if we think it's getting too big/convoluted to find things easily. I don't think we're there yet though.

So, could you just remove the comments please?